src/app/questions/questions.module.ts

```diff
@@ -12,14 +12,16 @@ import { FileComponent } from './file/file.component';
 import { TeamMemberSelectorComponent } from './team-member-selector/team-member-selector.component';
 import { FileDisplayComponent } from './file/file-display/file-display.component';
 import { FilestackModule } from '@shared/filestack/filestack.module';
+import { SharedModule } from '../shared/shared.module';
 
 @NgModule({
   imports: [
     IonicModule,
     CommonModule,
     FormsModule,
     ReactiveFormsModule,
-    FilestackModule
+    FilestackModule,
+    SharedModule
```

Comment:
Since we imported SharedModule, we don't need the IonicModule,CommonModule and FormsModule, because they are imported in the SharedModule already